resources/lang/en/people.php

```diff
@@ -151,14 +151,14 @@
     'information_edit_max_size' => 'Max :size Kb.',
     'information_edit_max_size2' => 'Max {size} Kb.',
     'information_edit_firstname' => 'First name',
-    'information_edit_lastname' => 'Last name (Optional)',
-    'information_edit_description' => 'Description (Optional)',
+    'information_edit_lastname' => 'Last name (optional)',
+    'information_edit_description' => 'Description (optional)',
     'information_edit_description_help' => 'Used on the contact list to add some context, if necessary.',
-    'information_edit_unknown' => 'I do not know this person’s age',
-    'information_edit_probably' => 'This person is probably…',
-    'information_edit_not_year' => 'I know the day and month of this person’s birthday, but not the year…',
-    'information_edit_exact' => 'I know the exact birthdate of this person…',
-    'information_edit_birthdate_label' => 'Birthdate',
+    'information_edit_unknown' => 'I do not know :name’s age',
+    'information_edit_probably' => ':name person is probably…',
```

Comment:
This message is used when you create a new contact. It's not possible to add the name of the contact.